backend/models/post.model.js

```diff
@@ -52,6 +52,31 @@ const postItem = new mongoose.Schema(
       max: 200,
       required: true,
     },
+    brand: { // brand of item (e.g Ikea, Wayfair...)
+      type: String,
+      max: 100,
+      required: true,
+    },
+    height: { // Height of item
+      type: String,
+      max: 100,
+      required: true,
+    },
+    width: { // Width of item
+      type: String,
+      max: 100,
+      required: true,
+    },
+    color: { // Color of item
+      type: String,
```

Comment:
So, when a user writes a color we will map the color to its rgba?

Reply:
bro just describes the color of the item brother